.github/workflows/ci.yml

```diff
@@ -33,23 +39,28 @@ jobs:
         run: |
           export CC="clang-9"
           export CXX="clang++-9"
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          default: true
+          components: "clippy, rustfmt, rust-src"
       - name: Check with Rustfmt
         uses: actions-rs/cargo@v1
         with:
           command: fmt
           args: --all -- --check
-      - name: Check with Clippy
-        uses: actions-rs/clippy-check@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-targets ${{ matrix.features }} -- -D warnings
+#       - name: Check with Clippy
+#         uses: actions-rs/clippy-check@v1
+#         with:
+#           token: ${{ secrets.GITHUB_TOKEN }}
+#           args: --all-targets ${{ matrix.features }} -- -D warnings
       - name: Build
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: ${{ matrix.features }} --verbose
+          args: ${{ matrix.features }} --verbose --target x86_64-unknown-linux-gnu -Zbuild-std
```

Comment:
why do we need to specify these build args?

Reply:
* `--target ...`: see https://github.com/rust-lang/rust/issues/48199 for some context. the last comment sorta explains it, but i don't really get it. effectively, building with sanitizers will fail without `--target` if a crate uses a build script or procedural macros
* `-Zbuild-std`: this rebuilds rust's standard library. it's not necessary for address sanitizers, so we can remove it here. but you need it for the memory sanitizer ([source](https://doc.rust-lang.org/beta/unstable-book/compiler-flags/sanitizer.html)) so it can accurately instrument all sources of allocations. but as long as we're using a prebuilt gdal in our tests here, allocations from gdal won't be instrumented so we can't do that anyways. in [some past runs in this pull request](https://github.com/georust/gdal/runs/1961853656), you can see how the sanitizer thinks there's a bunch of memory that's been leaked, presumably because gdal is not instrumented with sanitizers